target_chains/solana/programs/pyth-solana-receiver/Cargo.toml

```diff
@@ -24,11 +24,13 @@ wormhole-core-bridge-solana = {workspace = true}
 wormhole-raw-vaas = {version = "0.1.3", features = ["ruint", "on-chain"], default-features = false }
 pyth-solana-receiver-sdk = { path = "../../pyth_solana_receiver_sdk"}
 rand = "0.8.5"
+serde_with = { version = "2.3.3", default-features = false }
 
 [dev-dependencies]
-solana-sdk = { workspace = true }
+solana-sdk = { workspace = true } 
 tokio = "1.14.1"
 program-simulator = { path = "../../program_simulator" }
 wormhole-vaas-serde = { workspace = true }
 serde_wormhole = { workspace = true }
 common-test-utils = { path = "../../common_test_utils" }
+
```

Comment:
It doesn't compile

Reply:
https://solana.stackexchange.com/questions/15109/cargo-complaning-about-unmatchable-version-requirements-with-pyth-solana-receiv

Hey @guibescos I had this issue in Rust